CHANGELOG.md

```diff
@@ -24,13 +24,16 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.12.0...master
+[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.13.0...master
+
+## [3.13.0] - 2020-05-26
+[3.13.0]: https://github.com/atlassian/jira-actions/compare/release-3.12.0...release-3.13.0
 
 ### Added
-- Added top navigation bar handler to API - `com.atlassian.performance.tools.jiraactions.api.page.TopNav`
+- Add top navigation bar handler to API - `com.atlassian.performance.tools.jiraactions.api.page.TopNav`
 
 ### Fixed
-- Issue creation time will no longer include the dashboard loading time
+- Make issue creation time no longer include the dashboard loading time
```

Comment:
My OCD thanks you 🙂 
For changelog, I reuse language rules from Git commits: https://chris.beams.io/posts/git-commit/